### PR TITLE
Add support for decimal input type

### PR DESCRIPTION
**Pull request by npresseault**
## Description
This adds support for decimal number input type

## Motivation and Context
The number input type only allowed for integer

## How Has This Been Tested?
Added a sample and tested on android / iOS

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: android-ktx/build.gradle [hunk 1]
```diff
@@ -9,6 +9,7 @@ plugins {
 }
 
 repositories {
+    mavenLocal()
```
Comment (martingagnon) [inline android-ktx/build.gradle]: Leftover ? We try not to let this to prevent eventual errors.

**Comment by martingagnon** [comment]
Add a field in sample ?

File: android-ktx/build.gradle (hunk 1 above)
Comment (npresseault) [reply android-ktx/build.gradle]: Since we publish only snapshots in maven local why would that be a problem?

**Comment by npresseault** [comment]
This is already in sample

**Pull request merged**